package.json

```diff
@@ -2,7 +2,7 @@
   "name": "omnitone",
   "version": "0.2.0",
   "description": "Spatial Audio Decoder in Web Audio API",
-  "main": "src/main.js",
+  "main": "build/omnitone.js",
```

Comment:
I think we want `omnitone.min.js` instead. Is there any reason for you to prefer the unminified version?